src/agent.cpp

```diff
@@ -162,6 +165,14 @@ int Agent::getUnum() {
   return world().self().unum();
 }
 
+void Agent::setLastActionStatusCollision(bool may_fix, bool likely_success) {
```

Comment:
Thanks! Also, you're sure that collidesWithPost is the only condition that needs to be checked for? We don't need to account for collidesWithPlayer or collidesWithBall?